src/main/java/org/dsa/iot/knx/KnxIpConnection.java

```diff
@@ -159,9 +159,9 @@ private void connect() {
 		try {
 			if (serviceMode == KNXNetworkLinkIP.TUNNELING && null != remoteEP) {
 				networkLink = new KNXNetworkLinkIP(KNXNetworkLinkIP.TUNNELING, localEP, remoteEP, useNat,
-						new TPSettings(new IndividualAddress(deviceAddress), true));
+						new TPSettings(new IndividualAddress(deviceAddress)));
 			} else if (serviceMode == KNXNetworkLinkIP.ROUTING) {
-				networkLink = new KNXNetworkLinkIP(KNXNetworkLinkIP.ROUTING, localEP, null, useNat, TPSettings.TP1);
+				networkLink = new KNXNetworkLinkIP(KNXNetworkLinkIP.ROUTING, null, null, false, TPSettings.TP1);
```

Comment:
I just create separated class for tunneling and routing. Could you please have a look?